src/Microsoft.TestPlatform.AdapterUtilities/HierarchyConstants.cs

```diff
@@ -26,16 +26,42 @@ public static class Levels
         /// <summary>
         /// Total length of Hierarchy array.
         /// </summary>
-        public const int TotalLevelCount = 2;
+        /// <remarks>
+        /// Currently the order for this is usually:
+        /// <c>Managed Method Name</c>, <c>ClassName</c>, <c>Namespace</c>, <c>Assembly Display Name</c>.
+        /// </remarks>
+        public const int TotalLevelCount = 4;
```

Comment:
Is it ok to merge these breaking changes now?

Reply:
Yes, our contract with Visual Studio and adapters is changing.